grakn-bootup/src/main/java/ai/grakn/bootup/BootupProcessExecutor.java

```diff
@@ -42,7 +42,7 @@
 public class BootupProcessExecutor {
 
     public static final long WAIT_INTERVAL_SECOND = 2;
-    public static final String SH = "/bin/sh";
+    public static final String SH = "/usr/bin/env sh";
```

Comment:
@mgttlinger this change would not work because the program would try to find a binary named "env sh" rather than using the "sh" part as the argument to "env" correctly.

I would suggest creating two variables
```
public static final String ENV_BIN = "/usr/bin/env";
public static final String SHELL = "sh";
```
which will be used like so, from the other codes:
```
Arrays.asList(ENV_BIN, SHELL, "-c", "...")
```

Reply:
Also, have a look at the Jenkins / Travis test in the section below, it would help you.